.github/workflows/update-docker-compose.yml

```diff
@@ -18,14 +18,24 @@ permissions:
   contents: read
 
 jobs:
-  flow:
+  update-docker-compose:
     runs-on: ubuntu-22.04
     permissions:
       contents: write
+      pull-requests: write
     steps:
       - uses: actions/checkout@v4
 
+      - name: setup-git
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"  
+
       - name: simple-compose-service-updates
         uses: sbe-arg/simple-compose-service-updates@v1.1.0
+        with:
```

Comment:
We can also configure the thing to not create PRs at all but instead just run. Then I guess you need to check the run log to these if there are updates. Maybe better.. What do you guys think?

Reply:
I'd prefer to see PRs